packages/core/src/App/Components/Elements/Errors/error-component.jsx

```diff
@@ -19,6 +19,7 @@ const ErrorComponent = ({
     removeNotificationMessageByKey,
     type,
     is_iom,
+    is_eu,
```

Comment:
(throughout) This won't be the proper flag to look for, since it includes `maltainvest` clients as well. 

I suggest going for `standpoint.malta` as it is only exists on countries that provide that account.

Reply:
Ah yes, I think this is indeed nicer to use actually, I changed it to that. Although, I think it might not be a big issue. The notification will only show up if has_malta_account or has_iom_account is true (see client-notifications line 680 and trade-store line 1106). So if my assumption is correct, the user is only able to view the modal if he/she has a malta or iom account.